### PR TITLE
Fix #604 - replace marked.js with markdown-it.js

### LINKED ISSUE #604: Swap out marked.js for markdown-it
See https://github.com/markdown-it/markdown-it. Actively maintained and has a plugin architecture. :+1:

### PR DESCRIPTION
**Pull request by miketaylr**
Much nicer markdown library with plugin support. Once this is in place, we can add issue linking and pretty much anything else we want.

r? @magsout 

PS, let me know if you want me to deploy this to staging to play around with it. My local tests look good--and it fixes the problem of not being able to display more than one emoji in a comment and other crappy marked.js bugs. We can do that after #564 lands.

### CODE REVIEW

**Comment by miketaylr** [comment]
OK, deployed to staging.

**Comment by miketaylr** [comment]
Here's an issue you can test markdown out on: http://staging.webcompat.com/issues/241

File: webcompat/static/js/lib/issue-list.js [hunk 1]
```diff
@@ -5,6 +5,14 @@
 var issueList = issueList || {};
 issueList.events = _.extend({},Backbone.Events);
 
+if (!window.md) {
+  window.md = window.markdownit({
+    breaks: true,
+    html: true,
+    linkify: true
+  }).use(window.markdownitSanitizer);
```
Comment (miketaylr) [inline webcompat/static/js/lib/issue-list.js]: oops, missed the emoji plugin here.

**Comment by magsout** [comment]
r +

**Comment by magsout** [comment]
Faild here https://travis-ci.org/webcompat/webcompat.com/builds/57505689#L2838 But don't know why ..

**Comment by miketaylr** [comment]
Hmm weird.

**Comment by miketaylr** [comment]
I triggered a re-build, let's see how it goes.

**Comment by miketaylr** [comment]
Green now, let's merge. :confetti_ball:

**Comment by magsout** [comment]
:+1:

**Pull request merged**